packages/util/src/logger/cloud-sdk-logger.ts

```diff
@@ -203,6 +204,28 @@ export function getGlobalLogLevel(): string | undefined {
   return container.options.level;
 }
 
+/**
+ * Change the global transport of the container which will set default transport for all active loggers.
+ * e.g., to set the global transport call `setGlobalTransports(httpTransport)`.
+ * @param customTransports - The transport to set the global transport to. Both array and varargs are supported.
+ */
+export function setGlobalTransports(
```

Comment:
[pp] I know we did this method overloading in many places to allow for options:
 f(a)
f([a,b])
 and f(a,b). 

However, I find this overload hard and difficult to read. So for me a `TransportStream | TransportStream[]` to allow for a single or array object should be fine.

Reply:
I changed the function to just accept a single transport object or an array with multiple transports for readability.